.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+python:
+  - "2.7"
```

Comment:
Do we want to support 2.7, its going to be deprecated in a year.

Reply:
For the time being, we should, since many potential users could be using old or out-of-date operating systems.